articles/mfa/guides/configure-phone.md

```diff
@@ -0,0 +1,128 @@
+- Voice as an MFA factor is not available when using the [Classic Universal Login Experience](/universal-login/classic).
+::: 
+
+## End user experience - SMS
+
+![SMS End User 1](/media/articles/mfa/mfa-sms1.png)
+
+After signing up and entering a country code and phone number, the user will receive a six-digit code on their device, which they must enter to authenticate.
+
+![SMS End User 2](/media/articles/mfa/mfa-sms2.png)
+
+To use this option, your users must have a device capable of receiving text messages or phone calls. If users cannot receive them messages, they will not be able to authenticate with this factor. If they already enrolled this way and do not have such a device, they will need to use their recovery code to complete the MFA flow (and then enroll another MFA method in place of this one).
+
+## End user experience - SMS and Voice
+
+TBD
+
+## Administrative setup
+
+To allow users to authenticate with SMS or Voice, you must enable the Phone factor and select your preferred delivery method:
+
+* **Auth0**: Sends the messages using Auth0's internally-configured SMS delivery provider. It can be used for evaluation purposes, and there is a maximum of 100 messages per tenant during the entire tenant lifetime. You can't use this provider to send Voice messages.
+
+* **Twilio**: Sends the messages using the [Twilio Programmable SMS API](https://www.twilio.com/sms) for SMS or [Twilio Programmable Voice API](https://www.twilio.com/voice) for Voice. You will need to provide [your own Twilio credentials](#twilio-configuration). Make sure you use Twilio  **Live Credentials**, not the **Test Credentials**. The test credentials aren't meant to send messages for real.
+:::
+
+* **Custom**: Sends the messages by invoking the [Send Phone Message Hook](/hooks/extensibility-points/send-phone-message).
+
+Optionally, you can [customize your SMS or Voice notification templates](/mfa/guides/customize-phone-messages).
+
+![MFA SMS Settings](/media/articles/mfa/sms-settings.png)
+
+## Message Delivery Method
+
+You can choose if you want to give users the option of getting text messages, voice calls, or both.
+
+## Twilio configuration
+
+If you choose to deliver SMS via Twilio, follow these steps to configure your SMS factor.
+
+1. Open an account with Twilio. You will need a [Twilio Account SID](https://www.twilio.com/help/faq/twilio-basics/what-is-an-application-sid) and a [Twilio Auth Token](https://www.twilio.com/help/faq/twilio-basics/what-is-the-auth-token-and-how-can-i-change-it). These are the Twilio API credentials that Auth0 will use to send an messages to the user. 
+
+  You may also need to enable permissions for your geographic region for [SMS](https://support.twilio.com/hc/en-us/articles/223181108-How-International-SMS-Permissions-work) and [Voice](https://www.twilio.com/console/voice/calls/geo-permissions). If you use Voice, your Twilio phone number needs to be enabled to make Voice calls.
```

Comment:
Could we define a bit more what does it mean to have Twilio phone enabled? Is it about confirming the ownership of a specific phone number? I yes then I suggest to add a link to Twilio docs on the "enabled" word: https://support.twilio.com/hc/en-us/articles/223180048-Adding-a-Verified-Phone-Number-or-Caller-ID-with-Twilio